server/src/stacks/2020-10-01/stacks/function-app-stacks/Python.ts

```diff
@@ -1,146 +1,156 @@
 import { FunctionAppStack } from '../../models/FunctionAppStackModel';
+import { getDateString } from '../date-utilities';
 
-export const pythonStack: FunctionAppStack = {
-  displayText: 'Python',
-  value: 'python',
-  preferredOs: 'linux',
-  majorVersions: [
-    {
-      displayText: 'Python 3',
-      value: '3',
-      minorVersions: [
-        {
-          displayText: 'Python 3.10',
-          value: '3.10',
-          stackSettings: {
-            linuxRuntimeSettings: {
-              runtimeVersion: 'Python|3.10',
-              remoteDebuggingSupported: false,
-              isPreview: true,
-              isDefault: false,
-              isHidden: true,
-              appInsightsSettings: {
-                isSupported: true,
-              },
-              gitHubActionSettings: {
-                isSupported: true,
-                supportedVersion: '3.10',
-              },
-              appSettingsDictionary: {
-                FUNCTIONS_WORKER_RUNTIME: 'python',
-              },
-              siteConfigPropertiesDictionary: {
-                use32BitWorkerProcess: false,
-                linuxFxVersion: 'Python|3.10',
+const getPythonStack: (useIsoDateFormat: boolean) => FunctionAppStack = (useIsoDateFormat: boolean) => {
+  const python36EOL = getDateString(new Date(2021, 12, 23), useIsoDateFormat);
```

Comment:
Can we proactively add the dates for all versions?

```suggestion
  const python36EOL = getDateString(new Date(2021, 12, 23), useIsoDateFormat);
  const python37EOL = getDateString(new Date(2023, 06, 30), useIsoDateFormat);
  const python38EOL = getDateString(new Date(2024, 10, 31), useIsoDateFormat);
  const python39EOL = getDateString(new Date(2025, 10, 31), useIsoDateFormat);
  const python310EOL = getDateString(new Date(2026, 10, 31), useIsoDateFormat);
```

Reply:
done